code/modules/antagonists/ashwalker/ashwalker.dm

```diff
@@ -30,6 +30,7 @@
 /datum/antagonist/ashwalker/on_gain()
 	. = ..()
 	RegisterSignal(owner.current, COMSIG_MOB_EXAMINATE, .proc/on_examinate)
+	owner.current.mind.teach_crafting_recipe(/datum/crafting_recipe/skeleton_key)
```

Comment:
Isn't `owner.current.mind` the same as `owner`?

Reply:
This still needs looking at.

Reply:
no they arent